README.md

```diff
@@ -94,6 +94,28 @@ RSpec/FilePath:
     - spec/my_poorly_named_spec_file.rb
 ```
 
+## Non-goals of RuboCop RSpec
+
+### Enforcing `should` vs. `expect` syntax
+
+Enforcing:
```

Comment:
If you're going to make this read like a sentence, i suggest dropping this `:` as well as the one after "over" and making "Is" on the following line lowercase.